dags/process_elife_zip_dag.py

```diff
@@ -179,10 +179,10 @@ def wrap_article_in_libero_xml_and_send_to_service(**context) -> None:
 
     # add xml:base attribute to article element
     root = article_xml.getroot()
-    key = get_file_name_passed_to_dag_run_conf_file(context).replace('zip', '')
+    key = get_file_name_passed_to_dag_run_conf_file(context).replace('.zip', '')
```

Comment:
this should remove `.zip` from the end, rather than from anywhere; or get the name in any other way